functions/Get-DbaUserPermission.ps1

```diff
@@ -271,35 +312,8 @@ function Get-DbaUserPermission {
                             SourceView         = $row.'Source View'
                         }
                     }
-                }
-
-                Write-Message -Level Verbose -Message "Building data table for $db objects"
-                try {
-                    $dbDT = $db.Query($dbSQL)
                 } catch {
-                    # here to avoid an empty catch
-                    $null = 1
-                }
-
-                foreach ($row in $dbDT) {
-                    [PSCustomObject]@{
-                        ComputerName       = $server.ComputerName
-                        InstanceName       = $server.ServiceName
-                        SqlInstance        = $server.DomainInstanceName
-                        Object             = $db.Name
-                        Type               = $row.Type
-                        Member             = $row.Member
-                        RoleSecurableClass = $row.'Role/Securable/Class'
-                        SchemaOwner        = $row.'Schema/Owner'
-                        Securable          = $row.Securable
-                        GranteeType        = $row.'Grantee Type'
-                        Grantee            = $row.Grantee
-                        Permission         = $row.Permission
-                        State              = $row.State
-                        Grantor            = $row.Grantor
-                        GrantorType        = $row.'Grantor Type'
-                        SourceView         = $row.'Source View'
-                    }
+                    Write-Message -Level warning -Message "There was an error retrieving the permissions for $db on $instance."
```

Comment:
Why don't you use Stop-Function?

Reply:
As stated further up Stop-Function is now implemented.